_data/simulations/coral_6a/meta.yaml

```diff
@@ -0,0 +1,54 @@
+_id: 0dc3d1f0-76fd-11e7-a3e9-cb63094aa20f
+metadata:
+  author: Andrea Jokisaari
+  email: andrea.jokisaari@northwestern.edu
+  github_id: amjokisaari
+  simulation_name: coral_6a
+  summary: Upload-a-thon August 2017 test upload
+  timestamp: '31 July, 2017'
+  implementation:
+    repo:
+      url: 'https://www.dropbox.com/s/qusutz8g24hx54q/free_energy.csv'
```

Comment:
Should this link should use the ```dl``` subdomain instead of ```www```?
 https://dl.dropbox.com/s/qusutz8g24hx54q/free_energy.csv

Reply:
It does now

Reply:
Sorry, it's not important, the link makes no sense as it's just the link to the data

Reply:
"Just the link to the data?" That's, like, the most important thing on the page. But Dropbox still seems to be blocked for me, so I can't confirm that this works. Either merge it as-is and I'll check from my phone, or wait and I'll build at home this evening.